parsl/serialize/base.py

```diff
@@ -10,18 +11,24 @@ class SerializerBase:
     """ Adds shared functionality for all serializer implementations
     """
 
-    _identifier: bytes
-
-    @property
+    @cached_property
     def identifier(self) -> bytes:
-        """Get that identifier that will be used to indicate in byte streams
-        that this class should be used for deserialization.
-
+        """
+         Compute the identifier that will be used to indicate in byte streams
+￼        that this class should be used for deserialization.
+￼
+￼        Serializers that use identifiers that don't align with the way this is
+         computed (such as the default concretes.py implementations) should
+         override this property with their own identifier.
+￼
         Returns
```

Comment:
From the "not a big deal" camp, there's an extra space indent in this docstring.